backend/index.js

```diff
@@ -7,6 +7,9 @@ app.get('/', (req, res)=>{
     res.send('<h1>Hello Group</h1>');
 });
 
+// Buildings routes
+app.use('/buildings', require('./controllers/buildings'));
```

Comment:
Maybe change this line to:
`app.use('/api', routes);`

and at the beginning of the files add:
`const routes = require('./server/routes);`